src/style.css

```diff
@@ -1,100 +1,10 @@
 @import "./styles/normalize.css";
+@import "./styles/image.css";
```

Comment:
too much block-style files. you must keep all card__*** and stars__*** in separate files (same prefix to same prefix) `cause they are bem-blocks. and probobly you need 1-2 files to make it work correctly